### PR TITLE
Add page1180.txt

### PR DESCRIPTION
**Pull request by sebras**

### CODE REVIEW

File: transcribed/page1180.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+015     020 017 SUEXIT: JRST 2,@UUOH    ;USER UUO EXIT INS
+016             SURSTR: 0       ;LOC FOR NEW USER PC STORAGE AFTER UFLS
+017             SUUOH:  0       ;C(@41)
+018     001 018 SAC17P: MOVEM U,0       ;AC17 IN SHAD MEM
+019             AC14P:  0       ;PNTS TO AC 14 IN SHAD MEM
+020             AC15P:  0       ;"   ""    15
+021             AC16P:  0       ;  "   "   "" 16
+022             40P:    0       
+023             41P:    0       
+024             UPR:    0       ;USER PROTECTIONA AND RELOCATION REGISTERS
```
Comment (larsbrinkhoff) [inline transcribed/page1180.txt:24]: PROTECTION

File: transcribed/page1180.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+029             BUCSTP=200000   ;CORE STOPPING LOSER TO GIVE MORE CORE RQSTED BY ANOTHER LOSER
+030             BUSRC=100000    ;USER CONTROL INTERUPT TO SUPERIOR PROCEDURE
+031             BEXC=40000      ;SET BY UBO AND FRIENDS INHIBITS RELOCATION BY CORE ALLOCATOR
+032                     ;MUST BE SET WITH CLKOFF  MUST NOT BE SET IF CSTOP IS SET
+033             PIRQC:  0       ; PI REQ AWAITING SERVICE
+034             SUPPRO: 0       ;SUPERIOR PROCEDURE RH,INT BIT LH
+035             PICLR:  1       ;PI IN PROGRESS FLAG
+036             MSKST:  0       ;PI MASK REGISTER FOR USER
+037             FLSINS: 0       ;USER I-O WAIT INS
+038             ; STATES=> 0 => RUNABLE
```
Comment (larsbrinkhoff) [inline transcribed/page1180.txt:38]: STATES =>

**Review by larsbrinkhoff — CHANGES_REQUESTED** [review]

File: transcribed/page1180.txt (hunk 1 above)
Comment (sebras) [reply transcribed/page1180.txt:24]: Yes.

File: transcribed/page1180.txt (hunk 2 above)
Comment (sebras) [reply transcribed/page1180.txt:38]: Yup.

**Pull request merged**